pages/common/samtools.md

```diff
@@ -0,0 +1,28 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM file to BAM (use - for stdin):
+
+`samtools view -bS {{input.sam}} > {{output.bam}}`
```

Comment:
is the S an argument to the -b option, or a separate option? If so, what do those letters stand for?